### PR TITLE
Translate into Chinese

### PR DESCRIPTION
**Pull request by Mo-Dabao**
Some of the easy part has been translated into Chinese. The left is beyond me for now. I think Julia is exciting, would be catchy to MATLAB users. However, most of my work can be done using Python, witch has a more active society from my view, and I decide to focus on using Python for now. Best wishes for Julia.

### CODE REVIEW

**Comment by xorJane** [comment]
Thanks @Mo-Dabao for the translation!

@ViralBShah, I'm having a hard time adding @Roger-luo as a reviewer; it seems like I can only designate JC employees. Is there a way to work around that?

**Comment by ViralBShah** [comment]
I've invited @roger-luo to the org. Perhaps we can add him then.

**Comment by xorJane** [comment]
Thanks!

File: introductory-tutorials/intro-to-julia-ZH/01.新手入门.ipynb [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+   "source": [
+    "😺 = \"smiley cat!\"\n",
+    "typeof(😺)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "要打出“微笑的猫”，用tab键补全选择emoji的名字然后再按tab键就行【译注：在Jupyter中这么着好使，在REPL中不太好使】"
```
[the remark this answers is not in the document]
Comment (Mo-Dabao) [reply introductory-tutorials/intro-to-julia-ZH/01.新手入门.ipynb]: default
可能是系统不同的原因？我这REPL的猫贼丑。
 @Roger-luo 谢谢老哥的review，我翻译得不好都听老哥的。
这个红叉和红圈是干啥的呀？需要我做什么呢？需要我在哪里点个“同意修改”吗？

[the remark this answers is not in the document]
Comment (Roger-luo) [reply introductory-tutorials/intro-to-julia-ZH/01.新手入门.ipynb]: 你的系统REPL不支持emoji，所以出现乱码。

你需要自己重新修改我给你给出意见的地方。同时也建议你检查一下是否其它地方有类似的问题，这样我们可以快一点merge你的PR。

[the remark this answers is not in the document]
Comment (Mo-Dabao) [reply introductory-tutorials/intro-to-julia-ZH/01.新手入门.ipynb]: 我改过一波儿咯

**Comment by NHDaly** [comment]
@Roger-luo I'm trying to go through and clean up some of the old PRs in this repo. Do you think this PR is good to merge as-is? It seems better to have at least _something_ than to wait until it's perfect?

Or if you want to continue reviewing, please let me know! :)
I'll give this a few days and merge it if I don't hear from you (probably on Tuesday of next week)

**Comment by Roger-luo** [comment]
@NHDaly hi sorry I forgot to review this since last time, it's really an epic PR... but there was some translation notes were wrong in the part I reviewed, so I guess there might be some others part need to revise. 

It'd be nice if we could do it with small PRs @Mo-Dabao 

I'll try to spend some time reviewing it this month, and I could also post this on the Chinese discourse to see if people could help.

**Comment by ViralBShah** [comment]
cc @logankilpatrick too

**Comment by NHDaly** [comment]
If it's too much to merge, I'd also be supportive of just merging this and doing the improvements in small PRs over time, but totally up to y'all!

**Comment by NHDaly** [comment]
EDIT: s/merge/review/

**Comment by ViralBShah** [comment]
I would be ok with that

**Comment by Roger-luo** [comment]
yeah, that'd work too. I just created a post on Chinese discourse to ask for help: https://discourse.juliacn.com/t/topic/2884/2

but true, as you said maybe let's just merge this first.

**Comment by ViralBShah** [comment]
Probably should have a separate repo for the translations. What does everyone think?

**Comment by inkydragon** [comment]
I prefer to put those translations in a separate folder, not a new repo.

English files are located in the root directory, and Chinese translations are placed in the `zh-cn` folder.
Just like: [Learn X in Y minutes](https://github.com/adambard/learnxinyminutes-docs)

If we're going to open a new repo, we'd better wait until everything is translated.

**Comment by ViralBShah** [comment]
Ok. Makes sense. We can keep in the same repo.

**Comment by logankilpatrick** [comment]
Switched to use the file structure from "Learn X in Y minutes".

**Comment by logankilpatrick** [comment]
Going to merge unless there's last minute fixes required

**Pull request merged**